File: algorithm/typescript/BOJ2751.ts
// problem : https://www.acmicpc.net/problem/2751

import fs = require("fs");

const input: string[] = fs.readFileSync("/dev/stdin").toString().split("\n");
const N: number = Number(input[0]);
const result: number[] = [];

for (let i = 1; i <= N; i++) {
  result.push(Number(input[i]));
}

result.sort((a, b) => {
  return a - b;
});

console.log(result.join("\n"));

File: algorithm/python/BOJ2751.py
# problem : https://www.acmicpc.net/problem/2751

import sys

N = int(sys.stdin.readline().rstrip())
arr = list()

for i in range(0, N):
    arr.append(int(sys.stdin.readline().rstrip()))

arr.sort()

for i in range(0, N):
    print(arr[i])
